plugins/modules/elb_classic_lb_info.py

```diff
@@ -177,11 +355,30 @@ def describe_elb(connection, lb):
 
 @AWSRetry.jittered_backoff()
 def get_all_lb(connection):
+    """
+    Get paginated result for information of all Elastic Load Balancers.
+
+    Parameters:
+        connection (boto3.client): The Boto3 ELB client object.
+
+    Returns:
+        A list of dictionaries containing descriptions of all ELBs.
+    """
     paginator = connection.get_paginator("describe_load_balancers")
     return paginator.paginate().build_full_result()["LoadBalancerDescriptions"]
 
 
-def get_lb(connection, load_balancer_name):
+def get_lb(connection: Any, load_balancer_name: str) -> Union[Dict, List]:
```

Comment:
I guess you can import from `describe_load_balancers()` from  https://github.com/ansible-collections/amazon.aws/blob/main/plugins/module_utils/elb_utils.py#L109